android/cli/hooks/aar-transform.js

```diff
@@ -481,3 +446,96 @@ class SimpleFileCache {
 		fs.writeFileSync(this.cachePathAndFilename, dataToWrite);
 	}
 }
+
+/**
+ * Hyperloop versions below 2.2.0 have their own AAR handling which we need to
+ * disable by hooking into the affected hooks and reverting the changes Hyperloop
+ * made.
+ *
+ * @param {Object} cli CLI instance
+ * @param {Object} builder Builder instance
+ * @param {Object} logger Logger instance
+ */
+function registerHyperloopCompatibilityFixes(cli, builder, logger) {
+	var hyperloopModule = null;
+	builder.nativeLibModules.forEach(function (module) {
+		if (module.id === 'hyperloop' && appc.version.lt(module.version, '2.2.0')) {
+			hyperloopModule = module;
+		}
+	});
+	if (hyperloopModule === null) {
+		return;
+	}
+
+	var hyperloopBuildPath = path.join(builder.projectDir, 'build/hyperloop/android');
+
+	cli.on('build.android.aapt', {
+		priority: 1100,
+		/**
+		 * Remove parameters passed to AAPT which are not required anymore since 6.1.0
+		 *
+		 * @param {Object} data Hook data
+		 * @param {Function} callback Callback function
+		 */
+		pre: function(data, callback) {
```

Comment:
Our JavaScript coding style mandates there be a space between constructs and parenthesis. This rule applies to `function` and `while` as well as `if`.